client/src/routes/index.svelte

```diff
@@ -1,8 +1,33 @@
+<script context="module">
+	/** @type {import('@sveltejs/kit').Load<{
+	 * 		session: App.Session
+	 * }>} */
+	export async function load({ session }) {
```

Comment:
Does this type comment annotation thingy work for you? 
I still have type error: "Binding element 'session' implicitly has an 'any' type"

Reply:
It occured to me if I specify the lang attribute in `<script>` it might work and seems like it does, then import the types necessary. I also used an arrow function for `load` with the `Load` type..
![image](https://user-images.githubusercontent.com/37137309/158017562-cf46f133-5faf-4ca7-a713-364b0bb6b9a7.png)

Reply:
Good point, it was just copy-pasted from the docs, but I was fed up with it.
Will try your solution.

Reply:
Works much better now.